che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitStatusImpl.java

```diff
@@ -0,0 +1,263 @@
+    }
+
+    /** @see Status#setFormat(StatusFormat) */
+    @Override
+    public void setFormat(final StatusFormat format) {
+        this.format = format;
+    }
+
+    /** @see Status#getBranchName() */
+    @Override
+    public String getBranchName() {
+        return branchName;
+    }
+
+    /** @see Status#setBranchName(String) */
+    @Override
+    public void setBranchName(String branchName) {
+        this.branchName = branchName;
+    }
+
+    /** @see Status#getAdded() */
+    @Override
+    public List<String> getAdded() {
+        return added;
+    }
+
+    /** @see Status#setAdded(List) */
+    @Override
+    public void setAdded(List<String> added) {
+        this.added = added;
+    }
+
+    /** @see Status#getChanged() */
+    @Override
+    public List<String> getChanged() {
+        return changed;
+    }
+
+    /** @see Status#setChanged(List) */
+    @Override
+    public void setChanged(List<String> changed) {
+        this.changed = changed;
+    }
+
+    /** @see Status#getRemoved() */
+    @Override
+    public List<String> getRemoved() {
+        return removed;
+    }
+
+    /** @see Status#setRemoved(List) */
+    @Override
+    public void setRemoved(List<String> removed) {
+        this.removed = removed;
+    }
+
+    /** @see Status#getMissing() */
+    @Override
+    public List<String> getMissing() {
+        return missing;
+    }
+
+    /** @see Status#setMissing(List) */
+    @Override
+    public void setMissing(List<String> missing) {
+        this.missing = missing;
+    }
+
+    /** @see Status#getModified() */
+    @Override
+    public List<String> getModified() {
+        return modified;
+    }
+
+    /** @see Status#setModified(List) */
+    @Override
+    public void setModified(List<String> modified) {
+        this.modified = modified;
+    }
+
+    /** @see Status#getUntracked() */
+    @Override
+    public List<String> getUntracked() {
+        return untracked;
+    }
+
+    /** @see Status#setUntracked(List) */
+    @Override
+    public void setUntracked(List<String> untracked) {
+        this.untracked = untracked;
+    }
+
+    /** @see Status#getUntrackedFolders() */
+    @Override
+    public List<String> getUntrackedFolders() {
+        return untrackedFolders;
+    }
+
+    /** @see Status#setUntrackedFolders(List) */
+    @Override
+    public void setUntrackedFolders(List<String> untrackedFolders) {
+        this.untrackedFolders = untrackedFolders;
+    }
+
+    /** @see Status#getConflicting() */
+    @Override
+    public List<String> getConflicting() {
+        return conflicting;
+    }
+
+    /** @see Status#setConflicting(List) */
+    @Override
+    public void setConflicting(List<String> conflicting) {
+        this.conflicting = conflicting;
+    }
+
+    /** @see Status#getRepositoryState() */
+    @Override
+    public String getRepositoryState() {
+        return this.repositoryState;
+    }
+
+    /** @see Status#setRepositoryState(String) */
```

Comment:
you can just remove docs and java docs generator will use doc from parent method